scripts/govcms-deploy

```diff
@@ -84,3 +88,5 @@ else
   fi
 
 fi
+
+echo "Finished running govcms-deploy."
```

Comment:
another verbose output to show that the script has finished correctly and did not bail anywhere in the middle